<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="ITS0568.local" failures="1" tests="2" name="Default test" time="11.037" errors="0" timestamp="20 May 2020 15:55:54 GMT">
  <testcase classname="categoryNavigation.validateNavigationTest" name="mainPageNavValidation" time="10.986"/>
  <testcase classname="categoryNavigation.validateNavigationTest" name="pdpValidation" time="0.02">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method categoryNavigation.validateNavigationTest.pdpValidation() didn&amp;apos;t finish within the time-out 10">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method categoryNavigation.validateNavigationTest.pdpValidation() didn't finish within the time-out 10
]]>
    </failure>
  </testcase> <!-- pdpValidation -->
  <testcase classname="categoryNavigation.validateNavigationTest" name="filterValidation" time="0.0">
    <ignored/>
  </testcase> <!-- filterValidation -->
</testsuite> <!-- Default test -->
